src/pages/data-management/index.jsx

```diff
@@ -25,34 +27,46 @@ const DataManagementPage = ({ experimentId, experimentData, route }) => {
 
   const [projectsList, setProjectsList] = useState([]);
   const [newProjectModalVisible, setNewProjectModalVisible] = useState(true);
+  const [activeProjectIdx, setActiveProjectIdx] = useState(0);
 
   useEffect(() => {
     if (projectsList.length) {
       setNewProjectModalVisible(false);
     }
   }, [projectsList]);
 
-  const createNewProject = () => {
+  const createNewProject = (newProjectName) => {
+    const newProject = {
+      name: newProjectName,
+      createdDate: moment().local().format('DD MMM YYYY, HH:mm:ss [GMT]Z'),
+      lastModified: moment().local().format('DD MMM YYYY, HH:mm:ss [GMT]Z'),
```

Comment:
Shows date creation time according to local timezone. However, later on when the data is saved, it should be saved in UTC.

Reply:
Is this UTC task added to a ticket already?